### PR TITLE
[sc-71867] Having a workflow to increase the doc version

### PR DESCRIPTION
**Pull request by hoangmirs**
## What happened 👀

- Add an action to bump the version of Mulesoft documentation

## Insight 📝

Make the code as a composite action to use in multiple repositories

## Proof Of Work 📹

It's working on https://github.com/Jollibee-Foods-Corporation/jfc-global-exp-api-doc/pull/24

### CODE REVIEW

**Comment by tednguyendev** [comment]
As the current version in EXP API doc is `1.0.9`, I think the next version should be `1.0.10` instead of `1.1.1` like [this PR](https://github.com/Jollibee-Foods-Corporation/jfc-global-exp-api-doc/pull/25/files) a ✋ 

So to be sure, I just did this quick check 🙈:

I went to the [Design Center page](https://anypoint.mulesoft.com/designcenter/designer/#/project/ff3dc1df-b839-46b0-9475-f0e5272dfb4b) and try to Publish from Design Center to see how Mulesoft increase the version:

CleanShot 2023-10-09 at 13 09 48

And this `1.0.10` was auto-generated by Design Center:

CleanShot 2023-10-09 at 13 08 16

**Comment by hoangmirs** [comment]
> As the current version in EXP API doc is `1.0.9`, I think the next version should be `1.0.10` instead of `1.1.1` like [this PR](https://github.com/Jollibee-Foods-Corporation/jfc-global-exp-api-doc/pull/25/files) a ✋

Yes, it should be `1.0.10` or `1.1.0` or `2.0.0`, the current PR I just test with a random version and it should not be merged 🙏

**Review by tednguyendev — APPROVED** [review]

File: bump_version_doc/action.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    default: exchange.json
+  committer:
+    description: Committer for the pull request
+    required: true
+  assignees:
+    description: Assignees for the pull request
+    required: true
+  github_token:
+    description: GitHub token to create the pull request
+    required: true
```
Comment (andyduong1920) [inline bump_version_doc/action.yml:17]: should we move this into the `secret` instead of `input` @hoangmirs

**Review by andyduong1920 — COMMENTED** [review]
Should we add the usage into the README @hoangmirs

File: bump_version_doc/action.yml (hunk 1 above)
Comment (hoangmirs) [reply bump_version_doc/action.yml:17]: The composite action does not have the `secrets` attribute, all values will be passed as inputs 🙏

Comment (andyduong1920) [reply bump_version_doc/action.yml:17]: I see @hoangmirs

**Review by andyduong1920 — APPROVED** [review]

**Pull request merged**